lelkek.FrogBox/contact

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html charset="utf8">
+<html>
+<head>
+<title>FrogBox Store</title>
+<link rel="stylesheet" href="./css/web.css">
+<link rel="apple-touch-icon" sizes="57x57" href="./apple-icon-57x57.png">
+<link rel="apple-touch-icon" sizes="60x60" href="./apple-icon-60x60.png">
+<link rel="apple-touch-icon" sizes="72x72" href="./apple-icon-72x72.png">
+<link rel="apple-touch-icon" sizes="76x76" href="./apple-icon-76x76.png">
+<link rel="apple-touch-icon" sizes="114x114" href="./apple-icon-114x114.png">
```

Comment:
Is there a way to lump all of these together so you're not defining them all? Like links with the same rel element all in one object?